kik_unofficial/callbacks.py

```diff
@@ -10,6 +11,13 @@
 
 
 class KikClientCallback:
+    def _on_client_init(self, client=None):
```

Comment:
Why do we need `_on_client_init`?

Reply:
its the method that will save the client reference in the callback